.github/workflows/bump-version-and-update-milestone.yml

```diff
@@ -23,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: |
-          echo "Latest Version: ${{ github.event.inputs.version }}"
+          echo "Latest Version: ${{ inputs.version }}"
```

Comment:
Is it possible to move such update on input usage in Separate PR, to let me merge it sooner and test in manual execution of each action in next release.
It is huge milestone for us that release is fully in CI even there are few actions to run.

PR with workflow will take a bit more time to accept.

Reply:
You can reference same issue in commit, so it is pre-step fore functional PR.

Reply:
On it.

Reply:
Done.

Reply:
merged, please update this Pr